docs/api.yml

```diff
@@ -0,0 +1,226 @@
+openapi: "3.0.0"
+info:
+  title: RT Dashboard API
+  version: 1.0.0
+
+paths:
+  /history.json:
+    get:
+      summary: List history entries
+      responses:
+        200:
+          description: |
+            Return object containting list of rows and groups of history entries
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  rows:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        group:
+                          type: string
+                        subgroup:
+                          type: number
+                        start:
+                          type: number
+                        end:
+                          type: number
+                        type:
+                          type: string
+                          enum: [range, point]
+                        content:
+                          type: string
+                  groups:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        id:
+                          type: string
+                        content:
+                          type: number
+                        order:
+                          type: number
+  /job/{job_id}/cancel:
+    post:
+      summary: Cancel the job with the specified job_id
+      parameters:
+        - name: job_id
+          in: path
+          schema:
+            type: string
+            format: uuid
+          required: true
+      responses:
+        200:
+          description: Request processed
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+  /queue/{queue_name}/empty:
+    post:
+      summary: Empty the queue with the specified queue_name
```

Comment:
`[failed]` and `[finished]` and not queues but registries. Do you think we should mention that?

Reply:
i think this belongs in the usage docs, i also mentioned that failed/finished/runing are registries in the queues.json endpoint, i think this is enough for api docs